9c697d609329ca10c7067a418bba1022/update/sys_script_include_0bb58e71971112109790fefce053af28.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?><record_update table="sys_script_include">
+    <sys_script_include action="INSERT_OR_UPDATE">
+        <access>package_private</access>
+        <active>true</active>
+        <api_name>x_snc_plant_mgmt.get_plant_diseases</api_name>
+        <caller_access/>
+        <client_callable>false</client_callable>
+        <description>Once of script to get the plant diseases from an external source. Do not run this more than once otherwise duplicates will exist.</description>
+        <mobile_callable>false</mobile_callable>
+        <name>get_plant_diseases</name>
+        <sandbox_callable>false</sandbox_callable>
+        <script><![CDATA[var get_plant_diseases = Class.create();
+get_plant_diseases.prototype = {
+    initialize: function() {
+    },
+
+	getPlantDiseases: function(lat, lon) {
+        var diseases = new sn_ws.RESTMessageV2('Plant disease import', 'Get Plant Diseases');
+		var response = diseases.execute();
+		var responseBody = JSON.parse(response.getBody());
+		
+		// Get the data value from the returned payload and add to table.
+		for (var i = 0; i < responseBody.data.length; i++) {
+
+			var commonName = responseBody.data[i].common_name.toString();
+			var scientificName = responseBody.data[i].scientific_name.toString();
+			var family = '';
+			if (responseBody.data[i].family) {
+				family = responseBody.data[i].family.toString();
+			}
+			
+			for (var j = 0; j < responseBody.data[i].description.length; j++) {
+				var description = '';
+
+				description = responseBody.data[i].description[j].subtitle.toString() + '\n' + responseBody.data[i].description[j].description.toString() + '\n\n';
+
+			}
+
+			for (var k = 0; k < responseBody.data[i].solution.length; k++) {
+				var solution = '';
+
+				solution = responseBody.data[i].solution[k].subtitle.toString() + '\n' + responseBody.data[i].solution[k].description.toString() + '\n\n';
+				
+			}
+
+			for (var l = 0; l < responseBody.data[i].host.length; l++) {
+				var host = '';
+
+				host = responseBody.data[i].host[l] + ',';
+				
+			}
+
+			var diseaseRecord = new GlideRecord('x_snc_plant_mgmt_plant_diseases');
+			diseaseRecord.initialize();
+			diseaseRecord.common_name = commonName;
+			diseaseRecord.scientific_name = scientificName;
+			diseaseRecord.family = family;
+			diseaseRecord.description = description;
+			diseaseRecord.solution = solution;
+			diseaseRecord.host = host;
+			diseaseRecord.insert();
```

Comment:
Consider updating existing if values match such as common or scientific name so you avoid duplicates

Reply:
Hi @SapphicFire. I have updated the duplicate names to include the scientific name where the common name is the same, but the scientific is different. I also removed a genuine duplicate.